docker-compose.yml

```diff
@@ -12,11 +12,13 @@ services:
     environment:
       - PORT=4000
       - NODE_ENV=development
+    env_file: .env
```

Comment:
you added the `.env` configuration without adding file. The build is  failing if you merge it as is. either remove this line or add the file

Reply:
Wups, I did not mean to commit that. I will remove.

Reply:
Done. https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/24/commits/969d576677f1df660a3d1c703420fc181a598d51